mutaremalcolm/calorie_counter_app
# ## Project Setup for Calorie Counter App

### Description
This PR finalizes the initial project setup for the Calorie Counter app. It includes scaffolding the app with Vite, setting up Tailwind CSS for styling, installing Shadcn UI for components, and integrating Clerk for user authentication.

### Changes Implemented:
- Scaffolded the application using Vite.
- Installed and configured Tailwind CSS for styling.
- Integrated Shadcn UI for component management.
- Set up Clerk for handling user authentication.

### Next Steps:
- Build the user interface for tracking calorie input.
- Integrate the Calorie Calculator API for daily calorie recommendations.
- Develop authentication-related UI components.

### How to Test:
1. Clone the repository and navigate to the project directory.
2. Run `npm install` to install all dependencies.
3. Run `npm run dev` to start the development server.
4. Confirm that the app is running locally, and authentication works via Clerk.
